### PR TITLE
Update faq.svelte

### PR DESCRIPTION
**Pull request by LauraDuRy**
Updated the FAQ by removing two questions:
- When is pricing going to be enabled
- What will happen once pricing is enabled

Updated the FAQ by adding three questions:
- How to apply your credits
- Where to find usage overview
- Where to get all information on invoices and billing

## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/pricing/faq.svelte [hunk 1]
```diff
@@ -8,36 +8,44 @@
...
         },
         {
             question: 'Does Appwrite have a trial period?',
-            answer: 'Yes, we offer a 14-day trial period for you to explore Appwrite Pro. After the 14-day trial, you can choose to either subscribe to the Pro plan or continue on a Starter plan.'
+            answer: 'Yes, we offer a 14-day trial period for you to explore Appwrite Pro. After the 14-day trial, you can either subscribe to the Pro plan or continue on a Starter plan.'
         },
         {
             question: 'What payment methods does Appwrite support?',
-            answer: 'Appwrite currently supports credit and debit card payments. We will be working to constantly add support for more methods. Please reach out to us in case this is an issue for you.'
+            answer: 'Appwrite currently supports credit and debit card payments. We will be working to add support for more methods constantly. Please reach out to us in case this is an issue for you.'
```
Comment (adityaoberai) [inline src/routes/pricing/faq.svelte:20]: @LauraDuRy do we also want which card brands we allow, (Visa, Mastercard, Amex, Diners Club, etc.)?

File: src/routes/pricing/faq.svelte [hunk 2]
```diff
@@ -8,36 +8,44 @@
...
+        {
+            question: 'What happens if I reach a resource limit in my Pro plan?',
+            answer: 'Your project will continue to run, and additional charges will apply. You can find the costs for additional resources under the pricing plans comparison. We will also notify you when you hit 75% and 100% of your resources with per email. You can also use our budget caps to avoid unexpected payments.'
         },
         {
-            question: 'What happens if I reach the storage limit in my Pro plan?',
-            answer: 'Your project will continue to run, and additional charges will apply. You can find the costs for additional storage under the pricing plans comparsions. We will also notify you when you hit 75% and 100% of your storage with an alert in the dashboard and per email. You can also use our budget tools to avoid unexpected payments.'
+            question: 'What happens if I reach a resource limit in my Starter plan?',
+            answer: 'Your project will freeze, and you will be requested to upgrade to Pro or downgrade the resources used in your project. We will also notify you when you hit 75% and 100% of your resources with per email.'
```
Comment (adityaoberai) [inline src/routes/pricing/faq.svelte:28]: @LauraDuRy will this happen regardless of a budget cap, or they need to set one to get these emails at 75% and 100%?

File: src/routes/pricing/faq.svelte [hunk 3]
```diff
@@ -8,36 +8,44 @@
...
         },
         {
             question: 'How can I join the OSS program?',
             answer: 'The OSS program is exclusively for active open-source maintainers. In case you are liable for the program, you can <a href="/contact-us">apply here</a>.'
```
Comment (adityaoberai) [inline src/routes/pricing/faq.svelte:32]: Frontend issue but this link isn't easily visible as a link

image

Comment (adityaoberai) [reply src/routes/pricing/faq.svelte:32]: cc: @TGlide

**Review by adityaoberai — COMMENTED** [review]
@LauraDuRy have some comments

**Comment by adityaoberai** [comment]
@TGlide the Vercel deployment is fixed (`/kitchensink` was still around when the PR was made)

Any chance we can redeploy?

File: src/routes/pricing/faq.svelte (hunk 1 above)
Comment (LauraDuRy) [reply src/routes/pricing/faq.svelte:20]: Not here in the fAQ, but we would like to the docs. We will do this once docs is live.

File: src/routes/pricing/faq.svelte (hunk 2 above)
Comment (LauraDuRy) [reply src/routes/pricing/faq.svelte:28]: Yes, these emails you will get regardless.

Comment (adityaoberai) [reply src/routes/pricing/faq.svelte:28]: sounds good 👍

File: src/routes/pricing/faq.svelte (hunk 1 above)
Comment (adityaoberai) [reply src/routes/pricing/faq.svelte:20]: alright 👍

**Review by adityaoberai — APPROVED** [review]
LGTM 👍

**Pull request merged**